feat(HTTP): Refresh 2XX response pages (#34422)
mdn/content: files/en-us/web/http/status/207/index.md

> > **Note:** The ability to return a _collection of resources_ is part of the {{Glossary("WebDAV")}} protocol (it may be received by web applications accessing a WebDAV server).
> > Browsers accessing web pages will never encounter this status code.

Reviewer: - I think we can move this note to after the content paras, before the "Status" section. Moving the note to a later position would help readers to first understand the concept of the status code and then better relate with the context of the info in the note.
- We can also add more context in the main content to establish relation between 207 and WebDAV, so then the note will make more sense.
- The flow of info in the note itself can be improved:
```suggestion
> **Note:** Browsers accessing web pages will never encounter this status code. The ability to return a _collection of resources_ is part of the {{Glossary("WebDAV")}} protocol and is only encountered by web applications that access a WebDAV server.
> 
```

Author: done in ab929ff3e93203ef8107ddb50e930406ab61d9b9

> The HTTP **`207 Multi-Status`** status response code indicates that there might be a mixture of responses.

Reviewer: we can add the webdav context here:
```suggestion
The HTTP **`207 Multi-Status`** status response code indicates a mixture of responses. This response is used exclusively in the context of Web Distributed Authoring and Versioning ({{Glossary("WebDAV")}}).
```

It is part of the `200`-`299` class of [successful responses](/en-US/docs/Web/HTTP/Status#successful_responses).

The response body is a `text/xml` or `application/xml` HTTP entity with a `multistatus` root element that lists individual response codes.

## Status

```plain
207 Multi-Status
```

## Examples

> ### 207 Multi-Status response

Reviewer: how about
```suggestion
### Receiving a `207` response in a WebDAV context
```

> 

Reviewer: We can add a single line here that is a sample `207 Multi-Status` response from a WebDAV server. And anything else that can be called out/highlighted from the response.

Author: done in ab929ff3e93203ef8107ddb50e930406ab61d9b9
